src/types/onyx/ReportAction.ts

```diff
@@ -46,6 +46,9 @@ type ReportActionBase = {
     /** The ID of the reportAction. It is the string representation of the a 64-bit integer. */
     reportActionID?: string;
 
+    /** The ID of the previous reportAction on the report. It is a string represenation of a 64-bit integer (or null for CREATED actions). */
+    previousReportActionID?: string;
```

Comment:
As stated in the comment, there is a legitimate case for this to be null (i.e: CREATED actions).

Maybe a cleaner implementation would put `previousReportActionID: string` in the type for non-created actions and `previousReportActionID: null` in the type for CREATED actions, but right now this isn't configured that way.